modules/300-prometheus/docs/FAQ.md

```diff
@@ -371,7 +382,47 @@ spec:
 {% endofftopic %}
 
 After the resources deployment, Prometheus metrics will be available at address `lens-proxy/prometheus-lens-proxy:8080`.
-Lens Prometheus type - `Prometheus Operator`.
+Lens Prometheus type — `Prometheus Operator`.
+
+Due to some internal limitation of the Deckhouse platform, metrics on the Cluster and Nodes tabs will not be displayed.
```

Comment:
```suggestion
Starting from the version `???`, Lens requires `pod` and `namespace` labels to be present on node-exporter metrics. Otherwise, node resource consumption will not appear on Lens charts.
```

Reply:
i've fixed it